db/database.py

```diff
@@ -3,7 +3,7 @@
 from sqlalchemy.orm import sessionmaker
 from core.config import settings
 
-TEST_SQLALCHEMY_DATABASE_URL = "postgresql://postgres:postgres@localhost/st_crm_test_db"
```

Comment:
Pointing the test database to `settings.DATABASE_URL` will let it point to prod. So that's why I made it this way. Else we can't run tests easily. We'll find a better way of doing it later but if you take it out we'll forget.